src/query-builder/DeleteQueryBuilder.ts

```diff
@@ -73,10 +73,14 @@ export class DeleteQueryBuilder<Entity> extends QueryBuilder<Entity> implements
             const result = await queryRunner.query(sql, parameters);
 
             const driver = queryRunner.connection.driver;
-            if (driver instanceof MysqlDriver || driver instanceof AuroraDataApiDriver) {
+            if (driver instanceof MysqlDriver) {
                 deleteResult.raw = result;
                 deleteResult.affected = result.affectedRows;
 
+            } else if (driver instanceof AuroraDataApiDriver) {
+                deleteResult.raw = result;
+                deleteResult.affected = result.numberOfRecordsUpdated;
+
             } else if (driver instanceof SqlServerDriver || driver instanceof PostgresDriver || driver instanceof CockroachDriver) {
```

Comment:
You seems to fix the ``AuroraDataApiDriver`` thing, but you didn't change anything about this list of drivers. Is it normal? I don't know about the ``AuroraDataApiDriver``, but I know that this bug is available for ``PostgresDriver`` too and probably more drivers. Do the ``AuroraDataApiDriver`` fixes all?

Reply:
It obviously fixes only AuroraDataApiDriver (see the tag in the commit message). Would be nice to fix them all, unfortunately I do not have enough knowledge of the other drivers to submit a fix for everything.

This is a follow-up PR for the one that fixed MySQL driver, as explained in the issue.